services/QuillLMS/app/views/pages/shared/_teacher_center_navbar.html.erb

```diff
@@ -1,75 +1,23 @@
-<% @all = 'All' %>
-<% @research = 'Research' %>
-<% @faq = 'FAQ' %>
-<% @premium = 'Premium' %>
+<%-# locals: active_tab %>
+<%-# TODO - this feels like a bug, but everything is labeled mobile = false %>
+<% @mobile = false %>
```

Comment:
I don't think this is getting used at all anymore. It looks like `generic_tabs` takes @mobile as an argument, so I think it was previously getting passed in to be explicit that the mobile-only part wasn't supposed to show. I think just not passing it accomplishes the same thing, but then this line is unnecessary.

Reply:
Yeah, that's fair. This is equivalent code to what was there before, i.e. `@mobile = false` is set for all pages in this view. I'll remove this, since I don't see any code in the repo where `@mobile = true` is ever set (I was hesitant to remove in case the global var was set somewhere else, e.g. a layout file, meta code, etc. That's the problem with global settings).